.env.example

```diff
@@ -0,0 +1,2 @@
+SOURCECRED_DISCORD_TOKEN=
+SOURCECRED_GITHUB_TOKEN=
```

Comment:
suggest putting in dummy-but-realistic tokens here. (something with the right length but maybe all 0s?)